### PR TITLE
Report whether Windows licenses are baked into the firmware

### PR DESCRIPTION
**Pull request by wjt**
https://phabricator.endlessm.com/T18296

### CODE REVIEW

File: src/eos-metrics-instrumentation.c [hunk 1]
```diff
@@ -946,6 +971,33 @@ network_dbus_proxy_new (void)
 }
 
 static gboolean
+record_windows_licenses (gpointer unused)
+{
+    g_autoptr(GFile) tables = g_file_new_for_path (ACPI_TABLES_PATH);
+    g_auto(GVariantBuilder) builder;
+    gsize i;
+
+    g_variant_builder_init (&builder, G_VARIANT_TYPE ("a{sb}"));
```
Comment (ptomato) [inline src/eos-metrics-instrumentation.c]: Drive-by comment — consider using `bb` as the variant type so that we aren't storing hundreds of thousands of copies of the strings `SLIC` and `MSDM` in the database

Comment (rahmanShettia) [inline src/eos-metrics-instrumentation.c]: Although we would have  many of these strings a dictionary like {"MSDM": true, "SLIC": false} would be much feasible to data module as the rest of the dictionary like events

Comment (ramcq) [inline src/eos-metrics-instrumentation.c]: After discussions with @rahmanShettia we decided to go ahead with the self-documenting format on the basis that the metrics server will need to gain the ability to compress or normalise events as it scales in any case.

Comment (ptomato) [inline src/eos-metrics-instrumentation.c]: It also wastes user bandwidth, but I think we can get away with O(10 bytes) per user per boot
